interfaces/Matlab_Toolbox_Revamp/Class/Interface.m

```diff
@@ -0,0 +1,135 @@
+classdef Interface < handle & ThermoPhase & Kinetics
+    
+    properties
+        coverages
+    end
+    
+    methods
+        %% Interface class constructor
+
+        function s = Interface(src, id, p1, p2, p3, p4)
+            % parameter src:
+            %    CTI or CTML file containing the interface or edge phase.
+            % parameter id:
+            %    Name of the interface or edge phase in the source file.
+            % parameter p1/p2/p3/p4:
+            %    Adjoining phase to the interface;
+            % return:
+            %    Instance of class 'Interface'.
+            
+            checklib;
+            t = ThermoPhase(src, id);
+            s@ThermoPhase(src, id);
+            if nargin == 2
+                args = {};
+            elseif nargin == 3
+                args = {p1};
+            elseif nargin == 4
+                args = {p1, p2};
+            elseif nargin == 5
+                args = {p1, p2, p3};
+            elseif nargin == 6
+                args = {p1, p2, p3, p4};
+            end
+            s@Kinetics(t, src, id, args{:});
+            s.tp_id = t.tp_id;
+        end
+        
+        %% Interface methods
+                
+        function c = get.coverages(s)
+            % Get the surface coverages of the species on an interface.
+            %
+            % return:
+            %    If no output value is assigned, a bar graph will be
+            %    plotted. Otherwise, a vector of length "n_surf_species"
+            %    will be returned.
```

Comment:
I think the output value should be returned regardless. I don't see much utility in the "default" plot, and anyways, that's not what Python does. This same comment applies anywhere else this happened.